services/ui-src/src/views/NotFound/NotFound.tsx

```diff
@@ -1,8 +1,68 @@
 // components
-import { Flex, Text } from "@chakra-ui/react";
+import { Box, Flex, Heading, Image, Link, Text } from "@chakra-ui/react";
+// utils
+import { createEmailLink } from "utils/email/email";
+// assets
+import warningIcon from "../../assets/images/icon_warning.png";
+
+const cmsHelpEmail = "mdct_help@cms.hhs.gov";
 
 export const NotFound = () => (
-  <Flex h="100%" justifyContent="center" py="12">
-    <Text data-testid="404-view">Oops, page not found :( </Text>
-  </Flex>
+  <Box sx={sx.root} data-testid="404-view">
```

Comment:
Could move the verbiage here out to a data file like we've done for the other views. Same thing for the email variable above.